app/assets/images/book.svg

```diff
@@ -0,0 +1 @@
+<svg width="18" height="20" xmlns="http://www.w3.org/2000/svg"><path d="M18 2H4a2 2 0 100 4h14v13a1 1 0 01-1 1H4a4 4 0 01-4-4V4a4 4 0 014-4h13a1 1 0 011 1v1zM2 16a2 2 0 002 2h12V8H4a3.982 3.982 0 01-2-.535V16zM17 5H4a1 1 0 010-2h13v2z"/></svg>
```

Comment:
our glyphs should be 24 x 24px and the SVG tag should contain `viewBox` attribute as well.